README.rst

```diff
@@ -3,10 +3,22 @@ Tests
 
 Clear Containers Tests Repository
 
-Running functional tests
+Functional tests
 ------------------------
 
-To run functional tests::
+In order to run all functional test you have to install next packages
```

Comment:
you shouldn't need to do that, that's the purpose of vendoring. The git clone needs to be inside $GOROOT though.